packages/cerebral-demo/src/components/ClientForm/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import {connect} from 'cerebral/react'
+import Input from './Input'
+import Textarea from '../Textarea'
+
+export default connect(
+  {
+    client: `clients.$draft.**`
+  },
+  {
+    cancelEdit: 'clients.escPressed'
```

Comment:
Yes, disabled cancel edit on card click. Adding a TODO to notify on draft if changed.